src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2652,6 +2652,10 @@ private function registerCacheConfiguration(array $config, ContainerBuilder $con
                 }
             }
 
+            if ('cache.app' === $name && interface_exists(NamespacedPoolInterface::class)) {
```

Comment:
`cache.system` could also benefit from the fix, isn't it?

```suggestion
            } elseif (interface_exists(NamespacedPoolInterface::class)) {
```

Reply:
NVM! :facepalm: